tox.ini

```diff
@@ -31,6 +31,14 @@ deps =
 commands =
   stestr run {posargs}
 
+[testenv:rust]
+basepython = python3
+setenv =
+  PYTHONUSERBASE={envdir}
```

Comment:
The default tox configuration will build in release mode, I think for rust testing we should use debug mode by default.

Reply:
Done in 6856adbb9dc006ef0ed85f7869fd15778b470b0a.